packages/block-library/src/comment-template/edit.js

```diff
@@ -97,7 +102,7 @@ const CommentsList = ( {
 		{ comments &&
 			comments.map( ( comment ) => (
 				<BlockContextProvider
-					key={ comment.commentId }
+					key={ comment.commentId || uniqueId( 'comment-default' ) }
```

Comment:
This way, if we have a null id (required for loading placeholders and not calling REST API for comments), we don't have an error or warning regarding the unique key inside a list. Not sure if it is the best approach.

Reply:
It seems that this approach is preventing the block to be selected.

Reply:
I think we need to apply a fallback key as part of `comment` object to ensure those keys don't change. Otherwise, we will have unnecessary rerenders happening too often.

Reply:
Can we use the `comments.map()`'s `index` as a fallback?

```suggestion
					key={ comment.commentId || index }
```

It's only considered an antipattern if individual items can be removed or moved around, which shouldn't be the case for placeholders IIUC.